src/cljs/rems/status_modal.cljs

```diff
@@ -1,67 +1,118 @@
 (ns rems.status-modal
+  "Component for showing a status modal dialog.
+
+  There should only be one `status-modal` at the root of the component hieararchy.
+
+  Use the functions `set-pending!`, `set-success!` and `set-error!` to control its state.
+  See `rems.status-modal/component` for values to use in the calls."
```

Comment:
it's `rems.status-modal/status-modal`, right?

Reply:
Ah yes forgot to rename but I'll fix this reference instead for now.